modules/ui/commit_warnings.js

```diff
@@ -57,19 +54,19 @@ export function uiCommitWarnings(context) {
 
       let buttons = itemsEnter
         .append('button')
-        .on('mouseover', (d3_event, d) => {
-// todo replace legacy surface css class .hover
-          if (d.entityIds) {
-            const graph = editor.staging.graph;
-            context.surface().selectAll(utilEntityOrMemberSelector(d.entityIds, graph) )
-              .classed('hover', true);
-          }
-        })
-        .on('mouseout', () => {
-// todo replace legacy surface css class .hover
-          context.surface().selectAll('.hover')
-            .classed('hover', false);
-        })
+//        .on('mouseover', (d3_event, d) => {
```

Comment:
Should we just toast this block?

Reply:
I think we should replace it with the new way of hovering things - there still remain a bunch of places in the code that still uses the old "d3-select stuff from the surface and css class it" which won't work now.  We can do that in a separate pass though..